ui/packages/components/src/RunDetails/RunDetails.tsx

```diff
@@ -89,10 +90,10 @@ export function RunDetails({
       {run && history && getHistoryItemOutput && (
         <>
           <div className="px-5 pt-4">
-            {run.status && run.endedAt && run.output && isSuccess && (
+            {run.status && run.endedAt && run.output && (isSuccess || isFail) && (
```

Comment:
The `(isSuccess || isFail)` check isn't needed, since only completed or failed runs have an output